### PR TITLE
Add Sangfor Distributed File Storage(EDS) CSI driver to driver.md

### PR DESCRIPTION
**Pull request by evan37717**
Add Sangfor Distributed File Storage(EDS) CSI driver to driver.md
https://github.com/evan37717/sangfor-eds-csi

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [vSphere](https://github.com/kubernetes-sigs/vsphere-csi-driver) | `vsphere.csi.vmware.com` | v1.0 | A Container Storage Interface (CSI) Driver for VMware vSphere | Persistent | Read/Write Single Pod | Yes | Raw Block
 [YanRongYun](http://www.yanrongyun.com/) | ? | v1.0 | A Container Storage Interface (CSI) Driver for YanRong YRCloudFile Storage  | Persistent | Read/Write Multiple Pods | Yes | 
 [Zadara-CSI](https://github.com/zadarastorage/zadara-csi) | `csi.zadara.com` | v1.0, v1.1 | A Container Storage Interface (CSI) plugin for Zadara VPSA Storage Array & VPSA All-Flash  | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
-
+[Sangfor-EDS](https://github.com/evan37717/sangfor-eds-csi) | `eds.csi.sangfor.com` | v1.0 | A Container Storage Interface (CSI) Driver for Sangfor Distributed File Storage(EDS) | Persistent | Read/Write Multiple Pods | No |
```
Comment (saad-ali) [inline book/src/drivers.md]: Let's keep alphabetical order. Can you move this up the right spot.

**Comment by msau42** [comment]
You still seem to have some CLA issues. Can you make sure your commit is authored with the same email that you used to sign the CLA? (Also can you squash your commits?)

File: book/src/drivers.md (hunk 1 above)
Comment (evan37717) [reply book/src/drivers.md]: yes，i have move this up to the right spot, the commit ID: 425ef889c448041062b13c4fe14d17304f4bb098

**Comment by evan37717** [comment]
I signed it

**Comment by evan37717** [comment]
I have signed CLA,but cann't merge.the sign file as follow:













发件人：Michelle Au <notifications@github.com>
发送日期：2019-12-31 05:44:48
收件人：kubernetes-csi/docs <docs@noreply.github.com>
抄送人：evan37717 <heyun@sangfor.com.cn>,Mention <mention@noreply.github.com>
主题：Re: [kubernetes-csi/docs] Add Sangfor Distributed File Storage(EDS) CSI driver to driver.md (#247)
You still seem to have some CLA issues. Can you make sure your commit is authored with the same email that you used to sign the CLA? (Also can you squash your commits?)

**Comment by saad-ali** [comment]
Looks like CLA bot is happy now. Can you please rebase and squash your commits?

**Comment by evan37717** [comment]
i have rebase and squash the commits @saad-ali

**Comment by hoyho** [comment]
> i have rebase and squash the commits @saad-ali

you need to rebase it to 1 commit and solve conflict

**Pull request closed without merging**